R/labels.R

```diff
@@ -0,0 +1,102 @@
+#' Apply labels to multiple variables on a data frame
+#'
+#' This function allows a user to apply several labels to a dataframe at once.
+#'
+#' @param data A data.frame or tibble
+#' @param ... Named parameters in the form of variable = 'label'
+#'
+#' @return data with variable labels applied
+#'
+#' @importFrom rlang list2 is_named
+#' @importFrom purrr map_lgl walk2
+#' @export
+#'
+#' @examples
+#' add_labels(
+#'     mtcars,
+#'     mpg = "Miles Per Gallon",
+#'     cyl = "Cylinders"
+#'   )
+#'
+add_labels <- function(data, ...) {
+   # Pull out ellipsis to list
+   args <- rlang::list2(...)
+
+   # Check params
+   if (!inherits(data, 'data.frame')) stop("Labels must be applied to a data.frame or tibble")
```

Comment:
You don't have to change this, but it is nice for stuff like this to use stopifnot

Reply:
@statasaurus I guess a philosophical question. This would be user facing, so I like to customize and make those error messages explicit. `stopifnot()` won't let you do that. 

I can change it if you want, but that's my logic here. So let me know what you think.

Reply:
No I like your logic. I thought if you could name the expression and that would come out (it is the last example). But, no I don't really care. I never use stopifnot so I am really one to comment lol